### PR TITLE
feat: add ipfs.cyou

### PR DESCRIPTION
**Pull request by smurfix**
This is a well-connected server in Nürnberg Germany. Free bandwidth limited based on usage; faster paid access will be added later this year.

### CODE REVIEW

File: gateways.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [
+  "https://ipfs.cyou",
```
Comment (lidel) [inline gateways.json:2]: @smurfix I think you need to fix TLS setup. Currently, you have wildcard at wrong place:

> 

TLS wildcards work only for one level up. Due to this, you need to set up additional TLS wildcard certificates for DNS Names one subdomain higher: `*.ipfs.ipfs.cyou` and `*.ipns.ipfs.cyou`.

See https://docs.ipfs.tech/how-to/address-ipfs-on-web/#subdomain-gateway

You can reproduce with below:

- https://ipfs.cyou/ipfs/bafkreibm6jg3ux5qumhcn2b3flc3tyu6dmlb4xa7u5bf44yegnrjhc4yeq redirects to https://bafkreibm6jg3ux5qumhcn2b3flc3tyu6dmlb4xa7u5bf44yegnrjhc4yeq.ipfs.ipfs.cyou which then fails due to TLS error
- https://ipfs.cyou/ipns/en.wikipedia-on-ipfs.org redirects to https://en-wikipedia--on--ipfs-org.ipns.ipfs.cyou which then fails due to TLS error

**Review by lidel — CHANGES_REQUESTED** [review]

File: gateways.json (hunk 1 above)
Comment (smurfix) [reply gateways.json:2]: I'm already working on it. Unfortunately my DNS secondaries tend not to listen to update notifications.

Comment (smurfix) [reply gateways.json:2]: Fixed, now uses working wildcard certificates.

**Comment by lidel** [comment]
@smurfix ipfs.cyou returns 502 Bad Gateway errors. We will remove this gateway if not resolved in the near future.

**Comment by smurfix** [comment]
Thanks for the heads up. Will fix ASAP.

**Pull request merged**